panic: send on closed channel

```
panic: send on closed channel
goroutine 527 [running]:
github.com/kubernetes/dashboard/src/app/backend/handler.handleTerminalSession(0x1984000, 0xc0000dd360)
	/home/travis/build/kubernetes/dashboard/src/app/backend/handler/terminal.go:205 +0x43a
created by gopkg.in/igm/sockjs-go.v2/sockjs.(*handler).sockjsWebsocket
	/home/travis/gopath/pkg/mod/gopkg.in/igm/sockjs-go.v2@v2.0.1/sockjs/websocket.go:31 +0x337
```

##### Environment
Kubernetes version: v1.17.2
Dashboard version: kubernetesui/dashboard:v2.0.0-rc7


##### Steps to reproduce
Not sure, I think it was when navigating from a shell window to the pod list

--- a/go.mod
+++ b/go.mod
@@ -6,13 +6,12 @@ require (
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/emicklei/go-restful v2.12.0+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/igm/sockjs-go v2.0.1+incompatible // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/prometheus/client_golang v1.5.1
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 	golang.org/x/text v0.3.2
-	gopkg.in/igm/sockjs-go.v2 v2.0.1
+	gopkg.in/igm/sockjs-go.v2 v2.1.0
 	gopkg.in/square/go-jose.v2 v2.4.1
 	gopkg.in/yaml.v2 v2.2.8
 	k8s.io/api v0.18.1

